quantumdb-postgresql/src/main/java/io/quantumdb/core/schema/definitions/PostgresTypes.java

```diff
@@ -12,39 +13,55 @@
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class PostgresTypes {
 
-	public static ColumnType from(String input) {
```

Comment:
It's a bit hard to follow from outside of this class which method you need now: `fromString` or `from`. I would suggest the following to improve on this:

- Rename `fromString` back to `from`.
- Make the current `from(String)` private, and rename it to something like `valueOf(String)` which is typically used in enums (even though this isn't an enum).

This leaves 3 public methods:
- `from(String)`, which can return all types.
- `from(String, int)` which can return types with a scalar.
- `from(String, int, int)` which can return types with a scalar and precision.

This means the caller can pass what ever info they have (a single string, or a string with additional scalar / precision values).